src/less/components/search/SearchControl.less

```diff
@@ -0,0 +1,59 @@
+@import (reference) 'mixins';
+
+/*
+  Generic mixins to style text and text-like form controls
+  - <searcharea> elements
+  - all <input> elements, excluding:
+    - <input type="checkbox" />
+    - <input type="color" />
+    - <input type="radio" />
+    - <input type="range" />
+*/
+
+// input
+// searcharea
+.SearchControl(pristine) {
```

Comment:
@CITguy, let's discuss please.

We are using `.XxxxxControl` and `.XxxxxFacade` for various basic form control mixins, it would be best for us to be consistent.

Reply:
`.XxxxxControl` is for actual interactive controls, but we're styling the facade, here, so `.SearchFacade()` should be used instead.

Reply:
Although not currently an issue with `<hx-search>`, I think it would be better to consolidate all of the mixins (for styling the native element and facade) in one file, much in the same way that all of the Light DOM styles (layout and states) are consolidated.

Ok, I'll update to `.SearchFacade()`. 👍